src/main/java/pl/Lukasz/Calories/Entity/Users.java

```diff
@@ -0,0 +1,68 @@
+package pl.Lukasz.Calories.Entity;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name="users")
+public class Users {
+
+    private Integer userid;
+    private String username;
+    private String password;
+    private String role;
```

Comment:
User can only be in one role? If someone is an admin he cannot be a manager? One person should be able to have multiple roles